conf/nginx.conf

```diff
@@ -4,7 +4,7 @@ set $allowed_origins "https://${sandbox_domain}";
 set $api_domain "__DOMAIN__";
 set $files_domain "__DOMAIN__";
 ssl_ecdh_curve secp384r1;
-more_set_headers "Strict-Transport-Security: 'max-age=31536000; includeSubDomains' always";
+more_set_headers "Strict-Transport-Security: max-age=31536000; includeSubDomains; always";
 more_set_headers "X-XSS-Protection: '1; mode=block'";
 more_set_headers "X-Content-Type-Options: nosniff";
```

Comment:
Not super comfortable with those three lines :/ There's already https://github.com/YunoHost/yunohost/blob/dev/conf/nginx/server.tpl.conf#L52 and https://github.com/YunoHost/yunohost/blob/dev/conf/nginx/security.conf.inc#L33-L34

Reply:
redundant you think ?

Reply:
it needs to be tested, can you put this in another PR ? please